src/templates/IFunctionTemplate.ts

```diff
@@ -19,6 +19,7 @@ export interface IFunctionTemplate {
     language: string;
     isHttpTrigger: boolean;
     isTimerTrigger: boolean;
+    isSqlBindingTrigger: boolean;
```

Comment:
You also may want to consider refactoring this slightly to just be `templateType` or something - and then instead of having separate values like `isHttpTrigger` and `isTimerTrigger` you just have the type be `httpTrigger`, `timerTrigger`, `sqlBinding`, etc.

Reply:
I agree long-term, but since we're just trying to get this in soon with minimal impact, this seemed like the easiest solution.